alchemiscale/tests/integration/compute/client/test_compute_client.py

```diff
@@ -52,44 +52,49 @@ def test_api_check(
 
     ### compute
 
-    def test_query_taskqueues(
+    def test_query_taskhubs(
         self,
         scope_test,
         n4js_preloaded,
         compute_client: client.AlchemiscaleComputeClient,
         uvicorn_server,
     ):
-        taskqueue_sks = compute_client.query_taskqueues([scope_test])
+        taskhub_sks = compute_client.query_taskhubs([scope_test])
 
-        assert len(taskqueue_sks) == 2
+        assert len(taskhub_sks) == 2
 
-        taskqueues = compute_client.query_taskqueues([scope_test], return_gufe=True)
-        assert all([tq.weight == 0.5 for tq in taskqueues.values()])
+        taskhubs = compute_client.query_taskhubs([scope_test], return_gufe=True)
+        assert all([tq.weight == 0.5 for tq in taskhubs.values()])
 
-    def test_claim_taskqueue_task(
+    def test_claim_taskhub_task(
         self,
         scope_test,
         n4js_preloaded,
         compute_client: client.AlchemiscaleComputeClient,
         uvicorn_server,
     ):
-        taskqueue_sks = compute_client.query_taskqueues([scope_test])
-
-        # claim our first task
-        task_sks = compute_client.claim_taskqueue_tasks(taskqueue_sks[0], claimant="me")
-
-        # check that we got the task we expected given order
-        all_task_sks = n4js_preloaded.get_taskqueue_tasks(taskqueue_sks[0])
-
+        taskhub_sks = compute_client.query_taskhubs([scope_test])
+        # claim a single task; there is no deterministic ordering of tasks, so
+        # simply test that the claimed task is one of the actioned tasks
+        task_sks = compute_client.claim_taskhub_tasks(taskhub_sks[0], claimant="me")
+        all_task_sks = n4js_preloaded.get_taskhub_tasks(taskhub_sks[0])
         assert len(task_sks) == 1
-        assert task_sks[0] == all_task_sks[0]
-
-        # claim two more tasks
-        task_sks2 = compute_client.claim_taskqueue_tasks(
-            taskqueue_sks[0], count=2, claimant="me"
-        )
-
-        assert task_sks2 == all_task_sks[1:]
+        assert task_sks[0] in all_task_sks
+
+        # TODO: Currently we only allow tasks to be added to a taskhub with an EXTENDS
```

Comment:
This had to be modified due to #84 blocking adding of `:EXTENDS` tasks using `action_tasks`. Will be fixed in a separate PR.

Reply:
I added the first task in the chain for each of the three `transformations` in the `n4js_preloaded` fixture, so now we have multiple tasks to work with for this test.